src/Symfony/Component/Console/Tests/Input/ArgvInputTest.php

```diff
@@ -337,6 +337,21 @@ public function testHasParameterOptionEdgeCasesAndLimitations()
         $this->assertFalse($input->hasParameterOption('-fh'), '->hasParameterOption() returns true if the given short option is in the raw input');
     }
 
+    public function testNoWarningOnInvalidParameterOption()
+    {
+        $input = new ArgvInput(array('cli.php', '-edev'));
+
+        // Control.
+        $this->assertTrue($input->hasParameterOption(array('-e', '')));
+        // No warning is thrown if https://github.com/symfony/symfony/pull/26156 is fixed
+        $this->assertFalse($input->hasParameterOption(array('-m', '')));
+
+        // Control.
+        $this->assertEquals('dev', $input->getParameterOption(array('-e', '')));
+        // No warning is thrown if https://github.com/symfony/symfony/pull/26156 is fixed
+        $this->assertEquals('', $input->getParameterOption(array('-m', '')));
```

Comment:
should it really be an empty string here ? The option is not there, so it should be `false` AFAICT (this is the drawback of `assertEquals`: it performs type juggling)

Reply:
You are correct; that is an error in the test.